documentation/src/main/asciidoc/reference/search-dsl-predicate.asciidoc

```diff
@@ -630,6 +630,57 @@ but can be <<search-dsl-predicate-common-boost,boosted>>,
 either on a per-field basis with a call to `.boost(...)` just after `.field(...)`/`.fields(...)`
 or for the whole predicate with a call to `.boost(...)` after `.matching(...)`.
 
+[[search-dsl-predicate-terms]]
+== `terms`: match a series of term
+
+The `terms` predicate matches documents for which a given field contains some terms, any or all of them.
+
+With `matchingAny` we require that at least one of the provided terms matches.
+That is equivalent to having an <<search-dsl-predicate-boolean-or>> applied to the series of single-term predicates.
```

Comment:
This implies the boolean OR will contain `match` queries, in which case it's not exactly equivalent,  since we don't analyze terms.

What about this:

```suggestion
Functionally, this is somewhat similar to a <<search-dsl-predicate-boolean-or,boolean `OR`>> with one <<search-dsl-predicate-match,`match`>> predicate per term, but the syntax for a single `terms` predicate is more concise.

NOTE: `matchingAny` expects to be passed *terms*, not just any string. The given terms will not be analyzed/normalized. See <<search-dsl-predicate-terms-analysis>>.
```